File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) 
    {
        if(s.size()!=t.size()) return false;
        vector<int> v(26,0);
        for(int i=0;i<s.size();i++)
        {
            v[s[i]-'a']++;
            v[t[i]-'a']--;
        }    
        for(int i:v) if(i!=0) return false;
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        hmap={}
        if len(s)!=len(t):
            return False
        for i in s:
            if i in hmap:
                hmap[i]+=1
            else:
                hmap[i]=1
        for i in t:
            if i in hmap:
                hmap[i]-=1
            else:
                hmap[i]=1
        for i in hmap:
            if hmap[i]!=0:
                return False
        return True        